### ISSUE TITLE
Design fellowships directory wireframe for static directory

### ISSUE THREAD

**Issue by taisdesouzalessa** [mozilla/foundation.mozilla.org#934]
Context:
We need an MVP for the Fellowships directory page that considers the absence of the filtering feature since the information will be static at the beginning.

Some ideas on how to orient the users:
- divide in tabs (Science/Open Web...)
- alphabetical order
- chronological order (new cohorts first)

**Comment by taisdesouzalessa** [comment]
Amy asked us what type of info we need from the Fellows in order to provide content for this page. I think it is related to this issue since it influences the way this page will be designed, so I am posting it here.

According to feedback from Cori and Aurelia, the directory has to serve those 4 main cases:

**Prospective Fellows:**
1 ) Network factor: being a Fellow introduces them to a great network, which can also help with job searching
2) This shows prospective Fellows the caliber of people these fellowships draw.

**Current Fellows:**
3) They have been really interested in knowing more about the experience and expertise of other fellows

**Prospective Host Orgs:**
4) To point them to particularly stellar applicants who have already been through the program.

Matthew had suggested we user the same format as our People page, which is a good idea, since it satisfies most of the conditions above. I feel that are little parts missing, so I did a diagram.

Here is the diagram of the suggestion - the pink numbers represent the content that will support the goals of the page (the 4 main cases listed above).

<img width="986" alt="image" src="https://user-images.githubusercontent.com/22157921/34698868-42ff056e-f48f-11e7-9210-c2087138bf8f.png">

2 things to consider:
- I would recommend to add year besides the type of fellowship. Reason: to make it easy to recognize a cohort.
- It would be interesting to keep the role always updated (in this case, “Co-founder and CTO of...”) - because that reflects the advancement of fellows post-fellowship and also opportunities of network for prospective and past candidates.

@xmatthewx @jessevondoom could you please take a look and give me some feedback regarding the type of info each card should contain?  Once we define that, the goal is to do a spreadsheet similar to [this one that we had for People's page](https://docs.google.com/spreadsheets/d/1rwNFjojmnOHkctAIAin9Pd_rxziscGJBSWi-z2S6Bg4/edit?usp=sharing) so Amy and her team can fill in the content.

**Comment by xmatthewx** [comment]
Thanks for gathering the use case value. It'll really help us land a good MVP. 

To summarize @taisdesouzalessa, it sounds like you're suggesting: 
 - add a year field
 - require a short bio

Sounds great. Anything else?

**Note**: In form/spreadsheet, we should _require_ people to include a link to their Pulse profile (they should fill that out first). Then we can omit anything Pulse provides from this form/spreadsheet.

**Comment by taisdesouzalessa** [comment]
Good point on the Pulse requirement, I'll add that to the spreadsheet. 
Next steps (on my end): put the spreadsheet together and do the MVP wireframe.

**Comment by taisdesouzalessa** [comment]
@xmatthewx based on your last feedback, it seems the grey box won't exist in the MVP, correct? "See work" will connect straight to Pulse. So the Fellows short bio should be a requirement of Pulse, not of this site. Correct me if I am wrong, I got a bit confused on this one...
<img width="742" alt="image" src="https://user-images.githubusercontent.com/22157921/34787389-effb0526-f5eb-11e7-8545-56ab2adef772.png">

**Comment by taisdesouzalessa** [comment]
- Spreadsheet here: https://docs.google.com/spreadsheets/d/1fZXiQTn9BgGkWEDy7rSVabKHjbptbYDjUdKEMUygNt0/edit?usp=sharing

**Comment by taisdesouzalessa** [comment]
@xmatthewx @alanmoo here is the MVP of the directory - let me know what you think? Once I have the thumbs up I'll add these wireframes in the group document so other stakeholders can see it as well.

https://invis.io/QCFA50UP6#/272860083_It-5-Fellows_Directory_General
• Click on the areas and you'll see the smooth scrolling (I wanted to have all areas on the same initial page so it gives more unity to Fellowships)
• Click on "All Science Fellows" and see the specific Science directory (grouped by year so it is easier for current fellows to find folks from their cohort and connect, one of the use cases)

**Comment by xmatthewx** [comment]
Looks good. I'll let @alanmoo decide if this is mvp-ish enough. It's too bad we'll need to set up separate pages that we will later deprecate when a dynamic directory is ready, but seems necessary.

**Comment by alanmoo** [comment]
The main page feels like it makes sense for MVP, but I look at the "all Science fellows" and think that starts to really head towards "we need a way to manage these in a database of some sort" and it might take longer to do the back and forth of collecting all of those vs. just putting them into Pulse.

**Comment by taisdesouzalessa** [comment]
Hey @alanmoo does the second part of your sentence mean we should look into other design options (we can meet and investigate it together) or it is an observation to be aware of and we are proceeding with the page as is? Let me know how I can help.

**Comment by xmatthewx** [comment]
I also wonder how quickly the team will have content for prior cohorts. Those Area specific pages could just be CMS pages using the existing content about fellows currently posted on the advocacy and science sites. Name, photo, blurb. 

So maybe the main directory page follows @taisdesouzalessa's design, but includes all current fellows in each area. Then link to area hits a simple page with old info. Or... maybe those buttons lead to a single directory coming soon page and we can watch analytics to see if anyone clicks them.

**Comment by jessevondoom** [comment]
@xmatthewx Along the lines of your single-page / analytics idea. What if we focus on current fellows and give all previous cohorts more of a scaled back presence. We could just do a small headshot+name and any title/org information. 

My thinking here is we can require a Pulse profile for more info. It'll push those former fellows into Pulse with incentive (social links, project info, etc) and give us a lighter load for the time being. Not to mention it works toward a team goal of getting all fellows into Pulse.

**Comment by taisdesouzalessa** [comment]
Hey team, just to make sure I got this right, I did a quick mockup. This is what you mean @jessevondoom and @xmatthewx?
https://invis.io/WYFAPSXHX#/273044252_It-5-1-Fellows_Directory_General

I would prefer the solution of a simple page with old info if possible, with headshot+name and title/org info because that solution still works for the main use cases Cori and Aurelia mentioned in the doc. I think it is important we cater to those needs even in the MVP shape if we could.

I took a look at the past pages and so far we have:
10 fellows from 2015-2017 - Science
25 fellows from 2015-2017 - Open Web
11 (2017) - Tech Policy
??? - Media (I don't know)

We could manually include them in the CMS if necessary, for the MVP.

**Comment by alanmoo** [comment]
Sorry I wasn't entirely clear. My concern isn't really around the design, more of the time to implement the temporary solution vs the time it's going to take to gather and massage that content. It may end up that by the time the past content is structured properly, the API is ready.

**Comment by taisdesouzalessa** [comment]
Just to be sure we are on the same page, in my understanding I should use this MVP here 
https://invis.io/QCFA50UP6#/272860083_It-5-Fellows_Directory_General

Is this correct? @alanmoo @xmatthewx @jessevondoom

**Comment by xmatthewx** [comment]
@taisdesouzalessa - Yes, I think we work toward your plan, as designed. Summarizing all the thoughts on this thread, I propose we iterate like so: 

 1. build primary directory page as [designed](https://projects.invisionapp.com/share/QCFA50UP6#/screens/272860083_It-5-Fellows_Directory_General). include cards for all people in a current cohort. use headers separate fellowships by type. after that is done...
 2. add a button at bottom of each section. link to more for each fellowship type. depending on the state of content and engineering progress, @alanmoo can decide to link each more button to either: 
   - a cms page with name, pic, bio
   - designed page with cards for fellows
 3. when that is done, develop full directory

This focuses energy first on current cohort, and basic info for prior cohort. Then, all energy on full functional directory.
